acceptancetests/assess_caas_deploy_charms.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+
+    Spinning up k8s cluster and asserting the cluster is `healthy` - in this case, healthy means the k8s api server
+    is functionally working well, further health check needs to be done by charm deployment in next step
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+import os
+
+from deploy_stack import (
+    BootstrapManager,
+    deploy_caas_stack,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+    JujuAssertionError,
+    print_now,
+)
+
+from jujucharm import (
+    local_charm_path
+)
+
+__metaclass__ = type
+
+
+log = logging.getLogger("assess_caas_charm_deployment")
+
+
+def assess_caas_charm_deployment(client):
+    # Deploy k8s bundle to spin up k8s cluster
+    bundle = local_charm_path(
+        charm='bundles-kubernetes-core-lxd.yaml',
+        repository=os.environ['JUJU_REPOSITORY'],
+        juju_ver=client.version
+    )
+
+    caas_client = deploy_caas_stack(bundle_path=bundle, client=client)
+
+    if not caas_client.is_cluster_healthy:
+        raise JujuAssertionError('k8s cluster is not healthy coz kubectl is not accessible')
+
+    # add caas model for deploying caas charms on top of it
+    k8s_model = caas_client.add_model('testcaas')
+
+    gitlab_charm_path = local_charm_path(charm='caas-gitlab', juju_ver=client.version)
+    log.info('Deploying charm -> %s', gitlab_charm_path)
+    k8s_model.deploy(charm=gitlab_charm_path)
+
+    mysql_charm_path = local_charm_path(charm='caas-mysql', juju_ver=client.version)
+    log.info('Deploying charm -> %s', mysql_charm_path)
+    k8s_model.deploy(charm=mysql_charm_path)
+
+    k8s_model.juju('relate', ('gitlab', 'mysql'))
+    k8s_model.wait_for_workloads(timeout=3600)
+    print_now(caas_client.kubectl('get', 'all', '--all-namespaces'))
```

Comment:
Why are you printing this to stdout? If this is useful information we should log it. (also, this is actually the first time I've noticed the print_now function, I'm not sure why it's there instead of just relying on logging libs.)

Reply:
i thought we are using it. If not, I will just use log for all.